arrows/ocv/tests/test_resection_camera.cxx

```diff
@@ -0,0 +1,41 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include <arrows/ocv/resection_camera.h>
+
+#include <vital/plugin_loader/plugin_manager.h>
+
+#include <gtest/gtest.h>
+
+using namespace kwiver::vital;
+using namespace kwiver::arrows;
+using namespace std;
+
+using ocv::resection_camera;
+
+static constexpr double
+  ideal_rotation_tolerance = 1e-6,
+  ideal_center_tolerance = 1e-6,
+  noisy_rotation_tolerance = 0.008,
+  noisy_center_tolerance = 0.05,
+  outlier_rotation_tolerance = 0.008,
+  outlier_center_tolerance = 0.05;
+
+// ----------------------------------------------------------------------------
+int
+main( int argc, char** argv )
+{
+  ::testing::InitGoogleTest( &argc, argv );
+  return RUN_ALL_TESTS();
+}
+
+// ----------------------------------------------------------------------------
+TEST ( resection_camera, create )
```

Comment:
```suggestion
TEST( resection_camera, create )
```

Reply:
Huh... besides erroneously inserting the space here, uncrustify says:
```
do_space(3106): WARNING: unrecognize do_space:
   first->orig_line  is 1, first->orig_col  is 1, first->text()  'TEST', first->type is  FUNC_WRAP
   second->orig_line is 1, second->orig_col is 5, second->text() '(', second->type is PAREN_OPEN
   Please make a call at https://github.com/uncrustify/uncrustify/issues/new
```

...i.e. apparently it doesn't know how to format this. Therefore, I'm going to pretend I didn't see this for now... 🙂